docs/modules/lists/pages/continuation.adoc

```diff
@@ -1,11 +1,11 @@
 = List Continuation
 
+This section applies to complex lists—ordered or unordered lists that have multiple lines of text per item or that contain a mix of content types—that you want to keep continuous. 
+
 == Indented lines
 
-Aside from nested lists, all of the list items you've seen only have one line of text.
-But like with regular paragraph text, the text in a list item can wrap across any number of lines, as long as all the lines are adjacent.
-The wrapped lines can be indented, and they will still be treated as normal paragraph text.
-For example:
+As with regular paragraph text, the text in a list item can wrap across any number of lines as long as all the lines are adjacent. 
+Multiple lines are combined into a paragraph and wrap as normal paragraph text even if they're indented, as shown in the third bullet in this example:
```

Comment:
This explanation is much more concise :tada: Thanks!

Reply:
You're welcome, Sarah. I'm glad that the new phrasing works for you. I hope that others find it helpful too.